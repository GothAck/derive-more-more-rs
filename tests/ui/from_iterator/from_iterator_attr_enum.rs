use derive_more_more::FromIterator;

#[derive(FromIterator)]
enum Enum {
    Rar,
}

fn main() {}
