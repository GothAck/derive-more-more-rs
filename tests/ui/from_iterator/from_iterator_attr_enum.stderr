error: expected `struct`
 --> tests/ui/from_iterator/from_iterator_attr_enum.rs:4:1
  |
4 | enum Enum {
  | ^^^^
